### ISSUE TITLE
Commit Fask Mask code to repo

### ISSUE THREAD

**Issue by dadamsncsa** [nanoMFG/Face_Mask_tool#28]
##Description
Download the code from: https://www.pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/

copy to a subdirectory in `scr/`
